src/styles/reset.css

```diff
@@ -0,0 +1,6 @@
+* {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  box-sizing: border-box;
+  margin: 0;
```

Comment:
use `*` only for `box-sizing`

move font styles to `html` selector
move margin to some special tags you need to reset